frontend/src/components/messages/success-message/SuccessMessage.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import {useAppDispatch, useAppSelector} from "../../../app/hooks";
+import {selectProductSuccess, closeSuccess as closeProductSuccess} from "../../../slicer/productSlice";
+import {selectEmployeeSuccess, closeSuccess as closeEmployeeSuccess} from "../../../slicer/employeeSlice";
+import {selectOrderSuccess, closeSuccess as closeOrderSuccess} from "../../../slicer/orderSlice";
+import {selectSupplierSuccess, closeSuccess as closeSupplierSuccess} from "../../../slicer/supplierSlice";
+import {green} from "@mui/material/colors";
+//component imports
+import {
+    Avatar,
+    Button,
+    Card,
+    CardActions,
+    CardContent,
+    CardHeader,
+    Dialog,
+    DialogContent,
+    Typography
+} from "@mui/material";
+import CheckIcon from '@mui/icons-material/Check'
+//interface imports
+
+type Props = {};
+
+function SuccessMessage(props: Props) {
+    const product = useAppSelector(selectProductSuccess);
+    const employee = useAppSelector(selectEmployeeSuccess);
+    const order = useAppSelector(selectOrderSuccess);
+    const supplier = useAppSelector(selectSupplierSuccess);
+    const dispatch = useAppDispatch();
+    const close = () => {
+        dispatch(closeProductSuccess());
+        dispatch(closeEmployeeSuccess());
+        dispatch(closeOrderSuccess());
+        dispatch(closeSupplierSuccess());
+}
+    const open = [product, employee, order, supplier].some(x => !!x);
```

Comment:
Fancy expression :-) ( x => !!x): würde ich gerne verstehen. Sieht interessant aus

Reply:
some checkt, dass die bedingung fuer mindestens ein item des arrays zutrifft.
!!x heisst nicht nicht x.
also not not falsy in dem Fall